pkgs/build-support/builder-defs/builder-defs.nix

```diff
@@ -569,7 +569,7 @@ let inherit (builtins) head tail trace; in
      # Interpreters that are already in the store are left untouched.
          echo "patching script interpreter paths"
          local f
-         for f in $(find "${dir}" -xtype f -perm /0100); do
+         for f in $(find "${dir}" -xtype f ${findExecutablesFlags:--perm /0100}); do
```

Comment:
Oh damn, was not aware of this possibility, thanks for the hint, going to update this and try out all touched packages.